app/javascript/dashboard/routes/dashboard/settings/macros/MacrosTableRow.vue

```diff
@@ -1,59 +1,56 @@
-<script>
-import { useAccount } from 'dashboard/composables/useAccount';
+<script setup>
+import { computed } from 'vue';
 import Thumbnail from 'dashboard/components/widgets/Thumbnail.vue';
-export default {
-  components: {
-    Thumbnail,
-  },
-  props: {
-    macro: {
-      type: Object,
-      required: true,
-    },
-  },
-  setup() {
-    const { accountScopedUrl } = useAccount();
+import { useI18n } from 'dashboard/composables/useI18n';
 
-    return {
-      accountScopedUrl,
-    };
+const props = defineProps({
+  macro: {
+    type: Object,
+    required: true,
   },
-  computed: {
-    createdByName() {
-      const createdBy = this.macro.created_by;
-      return createdBy.available_name ?? createdBy.email ?? '';
-    },
-    updatedByName() {
-      const updatedBy = this.macro.updated_by;
-      return updatedBy.available_name ?? updatedBy.email ?? '';
-    },
-    visibilityLabel() {
-      return this.macro.visibility === 'global'
-        ? this.$t('MACROS.EDITOR.VISIBILITY.GLOBAL.LABEL')
-        : this.$t('MACROS.EDITOR.VISIBILITY.PERSONAL.LABEL');
-    },
-  },
-};
+});
+defineEmits(['delete']);
+const { t } = useI18n();
+
+const createdByName = computed(() => {
+  const createdBy = props.macro.created_by;
+  return createdBy.available_name ?? createdBy.email ?? '';
+});
+
+const updatedByName = computed(() => {
+  const updatedBy = props.macro.updated_by;
+  return updatedBy.available_name ?? updatedBy.email ?? '';
+});
+
+const visibilityLabel = computed(() => {
+  const i18nKey =
+    props.macro.visibility === 'global'
+      ? 'MACROS.EDITOR.VISIBILITY.GLOBAL.LABEL'
+      : 'MACROS.EDITOR.VISIBILITY.PERSONAL.LABEL';
+  return t(i18nKey);
+});
 </script>
 
 <template>
   <tr>
-    <td>{{ macro.name }}</td>
-    <td>
-      <div v-if="macro.created_by" class="avatar-container">
+    <td class="py-4 pr-4 truncate">{{ macro.name }}</td>
```

Comment:
@pranavrajs Let use optimize for RTL view also.
```suggestion
    <td class="py-4 ltr:pr-4 rtl:pl-4 truncate">{{ macro.name }}</td>
```